ansible_base/utils/transactions.py

```diff
@@ -0,0 +1,34 @@
+from django.db import connection
+
+
+def create_transaction(transaction_id):
+    cursor = connection.cursor()
+    cursor.execute(
+        "PREPARE TRANSACTION %s;",
+        [
+            str(transaction_id),
```

Comment:
Picky, but can we wrap these in a bit more protection? Something like `str(uuid(transaction_id))`. I'm worried about sql injection. Currently, I don't see a vector with their usage but I worry about anyone using them in the future.